codeworld-api/src/CodeWorld/Picture.hs

```diff
@@ -81,6 +81,11 @@ data Picture
           !Double
           !Double
           !Double
```

Comment:
I don't think you need this fourth parameter (the `w` or line width) to `Arc`.  It doesn't look like it's used anywhere.

Reply:
I removed the fourth parameter from ```Arc```. The width is used for ```ThickArc```? Or should I remove that as well? :/

Reply:
Line width is used for all thick shapes.